oracle-env.spec

```diff
@@ -1,7 +1,12 @@
 ### RPM cms oracle-env 31
 ## NOCOMPILER
 ## INITENV +PATH SQLPATH %i/etc
-## INITENV SET TNS_ADMIN %i/etc
+## INITENV SET TNS_ADMIN /etc
+
+### Alan on 16/Apr/2019
+# Instead of using an out-dated tnsnames.ora - shipped with this package - it assumes
+# the node has this RPM installed "oracle-instantclient-tnsnames.ora", which will sync
+# tnsnames from EOS/AFS to the localhost, under /etc/ directory
```

Comment:
@amaltaro , please remove `Source1: oracle-tnsnames.ora` and `cp %_sourcedir/oracle-tnsnames.ora* %i/etc/tnsnames.ora` lines too